v2/dtos/responses/event.go

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package responses
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/common"
+	dtoBase "github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/common/base"
+)
+
+// AddEventResponse defines the Response Content for POST event DTOs.  This object and its properties correspond to the
+// AddEventResponse object in the APIv2 specification.
+type AddEventResponse struct {
+	dtoBase.Response `json:",inline"`
+	ID               string `json:"id"` // ID uniquely identifies an event, for example a UUID
+}
+
+// Event represents a single measurable event read from a device
+type Event struct {
+	ID       string               `json:"id,omitempty" codec:"id,omitempty"`             // ID uniquely identifies an event, for example a UUID
```

Comment:
Other the `Modified`, how can any of these be empty?

Reply:
Remove all the unnecessary `omitempty` tags except for `Modified` and `Pushed` as suggested.